src/tmpfiles/tmpfiles.c

```diff
@@ -2557,12 +2557,10 @@ static int clean_item(Item *i) {
         case CREATE_SUBVOLUME_INHERIT_QUOTA:
         case CREATE_SUBVOLUME_NEW_QUOTA:
         case TRUNCATE_DIRECTORY:
-        case IGNORE_PATH:
         case COPY_FILES:
                 clean_item_instance(i, i->path);
                 return 0;
         case EMPTY_DIRECTORY:
-        case IGNORE_DIRECTORY_PATH:
```

Comment:
this goes directly against the description of 78a92a5a2306709e4587e332728a76901323ade9, no?

Reply:
Indeed, it does. I am/was confused as to what the purpose of the time-based cleanup for X was. I guess it's not exactly like `d` because it doesn't imply creation. It is almost the same as `e`, but `e` requires a time limit. IIUC, `X … -` should be equivalent to `e … 30y`.

So… I guess the best option would be to
a) move `X` closer to `d`/`D`/`e` in the list
b) adjust descriptions to clarify that it's close to `e`, except for the default age limit